packages/web/utils/questHelpers.ts

```diff
@@ -10,6 +10,25 @@ const { BN, amountToDecimal } = numbers;
 
 export const URIRegexp = /\w+:(\/\/)?[^\s]+/;
 
+export async function uploadQuestImage(file: File) {
+  const formData = new FormData();
+  formData.append('image', file);
+  const result = await fetch(`/api/storage`, {
+    method: 'POST',
+    body: formData,
+    credentials: 'include',
+  });
+
+  const response = await result.json();
+  const { error } = response;
+  if (result.status >= 400 || error) {
+    throw new Error(
+      `web3.storage ${result.status} response: "${error ?? result.statusText}"`,
+    );
+  }
+  return `ipfs://${response.image}`;
+}
+
```

Comment:
most of this functionality is already in `packages/web/utils/uploadHelpers.ts`. you could just reuse it like
```
import { uploadFile } from 'utils/uploadHelpers';
...
const ipfsHash = await uploadFile(file);
const url = `ipfs://${ipfsHash}`;
```

Reply:
Lol I was wondering why that didnt exist but I didnt check 🥴 thanks!

Reply:
Resolved as of [this commit]([41268b9](https://github.com/MetaFam/TheGame/pull/1448/commits/41268b977251b270e27babc89e05ce85333f0bb3))